src/script/miniscript.h

```diff
@@ -1148,8 +1209,8 @@ struct Node {
         return true;
     }
 
-    /** Return the maximum number of stack elements needed to satisfy this script non-malleably, including
-     * the script push. */
+    /** Return the maximum number of stack elements needed to satisfy this script non-malleably.
+     * This does not account for the P2WSH script push. */
```

Comment:
In a7db3f0503626c3671482720b22fedef7cd9e23c "miniscript: introduce a helper to get the maximum witness size"

nit: These comments should be changed in the previous commit.

Reply:
Done.